modules/imap_folders/modules.php

```diff
@@ -138,15 +138,15 @@ public function process() {
                 Hm_Msgs('ERRUnable to connect to the selected IMAP server');
                 return;
             }
-
             $specials = $this->user_config->get('special_imap_folders', array());
-            if ($exposed = $imap->get_special_use_mailboxes()) {
-                $specials[$form['imap_server_id']] = array('sent' => $exposed['sent'], 'draft' => '', 'trash' => $exposed['trash'], 'archive' => '', 'junk' => '');
-            } else if ($form['imap_service_name'] == 'gandi') {
-                $specials[$form['imap_server_id']] = array('sent' => 'Sent', 'draft' => 'Drafts', 'trash' => 'Trash', 'archive' => 'Archive', 'junk' => 'Junk');
```

Comment:
Seems like we had a special fix for gandi - maybe their expose list is empty and we need to hardcode the array? This PR removes the functionality - I wonder if it is on purpose?

Reply:
@kroky right. Their expose list is empty. I restored this functionnality